src/bundle.scss

```diff
@@ -0,0 +1,42 @@
+@forward "./normalize";
```

Comment:
You have to write a guide in `README.md` on how to run your project (watch scss).